CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# Changelog
+
+## [v3.0.0](https://github.com/voxpupuli/puppet-wildfly/tree/v3.0.0) (2023-10-27)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-wildfly/compare/v2.3.2...v3.0.0)
+
+**Implemented enhancements:**
+
+- Puppet unable to manage wildfly behind a secured connection [\#207](https://github.com/voxpupuli/puppet-wildfly/issues/207)
+- Initial install fails because /var/cache/wget does not exist. [\#191](https://github.com/voxpupuli/puppet-wildfly/issues/191)
+
+**Closed issues:**
+
+- Replace MaxPermSize with MaxMetaspaceSize in domain.conf template [\#288](https://github.com/voxpupuli/puppet-wildfly/issues/288)
+- Add parameters to domain.conf variables PROCESS\_CONTROLLER\_JAVA\_OPTS and HOST\_CONTROLLER\_JAVA\_OPTS [\#287](https://github.com/voxpupuli/puppet-wildfly/issues/287)
+- where the log file is stored on Puppet Agent by this module [\#260](https://github.com/voxpupuli/puppet-wildfly/issues/260)
+- JBOSS-EAP with \*\*systemctl start jboss-eap\*\*  don't works from another machine just  locally [\#258](https://github.com/voxpupuli/puppet-wildfly/issues/258)
+- Error: Could not set 'file' on ensure: No such file or directory @ dir\_s\_mkdir  [\#257](https://github.com/voxpupuli/puppet-wildfly/issues/257)
+- Wildfly 8.2.0 creating new jgroup stack [\#245](https://github.com/voxpupuli/puppet-wildfly/issues/245)
+- Support for Wildlfy 11 [\#230](https://github.com/voxpupuli/puppet-wildfly/issues/230)
+
+**Merged pull requests:**
+
+- Update Readme code examples and rubocop\_todos [\#301](https://github.com/voxpupuli/puppet-wildfly/pull/301) ([rwaffen](https://github.com/rwaffen))
+- modulesync 7.0.0-10-g003fb9c [\#300](https://github.com/voxpupuli/puppet-wildfly/pull/300) ([rwaffen](https://github.com/rwaffen))
```

Comment:
that one has the modulesync label, it shouldnt appear here 🤔

Reply:
i think thats my falt, i cheated and ran the ghcg manually, because with the rake task it always broke with "cannot find tag 2.3.2". even if i specified the since-tag with v2.3.2 :(

Reply:
probably because the release line in the history.md is missing

Reply:
the v's in front of the tag name in history.md where missing